.github/workflows/apk-s3-distribute.yml

```
@@ -0,0 +1,34 @@
+name: App Distribute CI
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  build_ui_components_sample_app:
+    name: Build and push to S3
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Prepare environment
+        run: |
+          git fetch --unshallow
+          echo "${{ secrets.RELEASE_KEYSTORE }}" > .sign/release.keystore.asc
+          gpg -d --passphrase "${{ secrets.PASSPHRASE }}" --batch .sign/release.keystore.asc > .sign/release.keystore
+          echo "${{ secrets.RELEASE_KEYSTORE_PROPERTIES }}" > .sign/keystore.properties.asc
+          gpg -d --passphrase "${{ secrets.PASSPHRASE }}" --batch .sign/keystore.properties.asc > .sign/keystore.properties
+          echo "AWS_S3_BUCKET=${{ secrets.AWS_S3_BUCKET }}" >> $GITHUB_ENV
+      - name: Assemble
+        run: bash ./gradlew :stream-chat-android-ui-components-sample:assembleDemoRelease --stacktrace
```

This is not needed, we don't use it in our other workflows

```suggestion
        run: ./gradlew :stream-chat-android-ui-components-sample:assembleDemoRelease --stacktrace
```